crates/runtime/tests/runtime_tests.rs

```diff
@@ -136,27 +136,25 @@ fn memory_runtime_validate_deploy_ok() {
 }
 
 #[test]
-fn memory_runtime_validate_spawn_missing_template_addr() {
+fn memory_runtime_validate_spawn_eof() {
     let runtime = testing::create_memory_runtime();
     let message = vec![0xFF, 0xFF];
 
-    let error = ParseError::NotEnoughBytes(Field::Address);
-    let expected = ValidateError::Parse(error);
-
-    let actual = runtime.validate_spawn(&message).unwrap_err();
-    assert_eq!(expected, actual);
+    assert!(matches!(
+        runtime.validate_spawn(&message),
+        Err(ValidateError::Parse(ParseError::Eof))
```

Comment:
The previous version of the test had a different error, it said what field was missing bytes